admin/views/service-manager/upload-pupil-census.ejs

```diff
@@ -0,0 +1,45 @@
+<div class="grid-row">
+    <div class="column-one-half">
+        <%- partial('partials/_breadcrumbs', { breadcrumbs: breadcrumbs }) %>
+    </div>
+    <div class="column-one-half">
+        <%- partial('partials/_name_and_sign_out') %>
+    </div>
+</div>
+
+<header class="page-header">
+    <h1 class="heading-xlarge">Upload pupil census</h1>
+</header>
+
+<div class="column-two-thirds">
```

Comment:
Shouldn't this be wrapped in a `<div class="grid-row">`
https://govuk-elements.herokuapp.com/layout/

Reply:
@jon-shipley i missed this comment.
I added the missed element.